src/modules/normalize/filter_audiolevel.yml

```diff
@@ -28,6 +28,14 @@ parameters:
     default: 1
     widget: checkbox
 
+  - identifier: peak
+    title: output samples peak value in dB instead of percentage in _audio_level.<N>
+    type: integer
+    minimum: 0
+    maximum: 1
```

Comment:
Use `boolean` here and get rid of minimum and maximum. To do so, you need to change the `schema_version` above to "7.0". And please update the `iec_scale` parameter to match.